### PR TITLE
Review public psy profile title & description

### PR DESCRIPTION
**Pull request by sandrica89**
... and add some spacing

May conflict with #322

### CODE REVIEW

File: frontend/src/components/PsyListing/PublicPsychologistProfile.jsx [hunk 1]
```diff
@@ -81,39 +81,54 @@ const PublicPsychologistProfile = () => {
...
-            <div key={field.name} className={styles.field} data-test-id="psy-info">
-              <div className={styles.fieldName}>
-                {field.name}
+        <>
+          <Row>
+            <div key="name" className={styles.field} data-test-id="psy-name">
+              <div className={styles.psyName}>
+                {psychologist.lastName.toUpperCase()}
+                {' '}
+                $
```
Comment (desoindx) [inline frontend/src/components/PsyListing/PublicPsychologistProfile.jsx:90]: extra $

File: frontend/src/components/PsyListing/PublicPsychologistProfile.jsx [hunk 2]
```diff
@@ -81,39 +81,54 @@ const PublicPsychologistProfile = () => {
...
-        <Col n="md-6 sm-12">
-          {fields.map(field => (
-            <div key={field.name} className={styles.field} data-test-id="psy-info">
-              <div className={styles.fieldName}>
-                {field.name}
+        <>
+          <Row>
+            <div key="name" className={styles.field} data-test-id="psy-name">
+              <div className={styles.psyName}>
+                {psychologist.lastName.toUpperCase()}
```
Comment (desoindx) [inline frontend/src/components/PsyListing/PublicPsychologistProfile.jsx:88]: If you can use a string template ?

File: frontend/src/components/PsyListing/publicPsychologistProfile.cssmodule.scss [hunk 3]
```diff
@@ -1,13 +1,21 @@
+.psyName {
+  font-size: 24px;
+  font-weight: bold;
+  padding-bottom: 12px;
```
Comment (desoindx) [inline frontend/src/components/PsyListing/publicPsychologistProfile.cssmodule.scss:4]: can you use multiple of 8 for margin and padding ?

File: frontend/src/components/PsyListing/publicPsychologistProfile.cssmodule.scss [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
 
 .fieldName {
   font-weight: bold;
+  padding-bottom: 5px;
 }
 
 .mapContainer {
-    width: 100%;
+  width: 100%;
+  margin-top: 12px;
```
Comment (desoindx) [inline frontend/src/components/PsyListing/publicPsychologistProfile.cssmodule.scss:18]: why do you need a margin top ?

**Review by desoindx — CHANGES_REQUESTED** [review]

File: frontend/src/components/PsyListing/publicPsychologistProfile.cssmodule.scss (hunk 4 above)
Comment (sandrica89) [reply frontend/src/components/PsyListing/publicPsychologistProfile.cssmodule.scss:18]: This margin-top is to add space between fields and map on mobile view:
Selection_082

Comment (desoindx) [reply frontend/src/components/PsyListing/publicPsychologistProfile.cssmodule.scss]: then you can use the gap on display flex https://developer.mozilla.org/fr/docs/Web/CSS/gap

**Review by desoindx — APPROVED** [review]

**Pull request merged**